### PR TITLE
fix(i18n): Portuguese testStrings translated

### PR DESCRIPTION
**Pull request by PeterF521**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

I did some translations on testStrings and I updated slice and splice description.

### CODE REVIEW

File: curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/basic-algorithm-scripting/slice-and-splice.portuguese.md [hunk 1]
```diff
@@ -21,17 +21,17 @@ localeTitle: Fatia e Splice
...
-  - text: Todos os elementos da primeira matriz devem ser adicionados à segunda matriz na ordem original.
-    testString: 'assert.deepEqual(frankenSplice([1, 2, 3, 4], [], 0), [1, 2, 3, 4], "All elements from the first array should be added to the second array in their original order.");'
+    testString: 'assert.deepEqual(frankenSplice(["claw", "tentacle"], ["head", "shoulders", "knees", "toes"], 2), ["head", "shoulders", "claw", "tentacle", "knees", "toes"], "<code>frankenSplice(["claw", "tentacle"], ["head", "shoulders", "knees", "toes"], 2)</code> deve retornar <code>["head", "shoulders", "claw", "tentacle", "knees", "toes"]</code>.");'
+  - text: Todos os elementos do primeiro array devem ser adicionados à segunda matriz na ordem original.
+    testString: 'assert.deepEqual(frankenSplice([1, 2, 3, 4], [], 0), [1, 2, 3, 4], "ATodos os elementos do primeiro array devem ser adicionados à segunda matriz na ordem original.");'
```
Comment (nhcarrigan) [inline curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/basic-algorithm-scripting/slice-and-splice.portuguese.md:30]:
```suggestion
    testString: 'assert.deepEqual(frankenSplice([1, 2, 3, 4], [], 0), [1, 2, 3, 4], "Todos os elementos do primeiro array devem ser adicionados à segunda matriz na ordem original.");'
```
This appears to be a typo?

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Looks like a slight typo. 

Additionally, can you explain the decision to change the challenge description? It appears your translation deviates from the English version.

**Comment by RandellDawson** [comment]
FYI - Only a test's `text` will be displayed to a user.  The assert message argument is ignored, so translating it is not necessary.  Just make sure the `text` is the correct translation of the current English version on the `master` branch.

**Review by RandellDawson — DISMISSED** [review]
LGTM

**Comment by RandellDawson** [comment]
@nhcarrigan Are the changes in this PR not warranted?  If the challenge description is not the same as the English, then I think we can close this PR.  Just let me know.

**Comment by nhcarrigan** [comment]
Heya @RandellDawson,

In full disclosure, I studied Spanish - lexically very similar to Portuguese, but there may be some idiomatic differences. 

The current English text is:
>You are given two arrays and an index. Copy each element of the first array into the second array, in order. Begin inserting elements at index n of the second array. Return the resulting array. The input arrays should remain the same after the function runs.

The *original* translation is roughly:
>You are given two arrays and an index. Use the `slice` and `splice` array methods to copy each element from the first array to the second array, in order. Start by inserting elements into the `n`th index of the second array. Return the resulting array. The input arrays must remain the same after the function is executed. Remember to use Read- Search-Ask if you get stuck. Write your own code.

The proposed new translation is roughly: 
> Copy each element from the first array to the second array, in order. Start by inserting elements into the `n`th index of the second array. Return the resulting array. The input arrays must remain the same after the function is executed. Remember to use the Read-Search-Ask if you get stuck. Write your own code.

I'm not sure if the English text was updated since I last looked at this - but it seems the new translation is closer to the English text than the original translation, now. Neither are exact matches, however.

**Comment by RandellDawson** [comment]
@PeterF521 Can you please take a look at the latest English version of this challenge and update this PR to better match the English version?

https://github.com/freeCodeCamp/freeCodeCamp/blob/master/curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-algorithm-scripting/slice-and-splice.english.md

Thanks in advance.

**Comment by PeterF521** [comment]
Ok, I'll do it

**Review by raisedadead — APPROVED** [review]

**Pull request merged**